Common/Data/Auxiliary/MapFileResolver.cs

```diff
@@ -148,6 +148,7 @@ public MapFile ResolveMapFile(string symbol, DateTime date)
                     {
                         // the searched date is greater than the last date in the list, return the last entry
                         indexOf = entries.Keys.Count - 1;
+                        if (indexOf == 0) return null;
```

Comment:
Not sure this is right 🤔 - For the added unit test, add for example `new MapFileRow(new DateTime(2005, 12, 31), "amz")` and should still return `null`, but it doesn't, right?
I think it should directly return `null` in this `if` case, see comment on line 149

Reply:
> add for example new MapFileRow(new DateTime(2005, 12, 31), "amz") and should still return

Rows in map files with duplicated tickers only makes sense in the first row (listing date) or in the last row (deslisting date or if it is still listed, 2050-12-31).

Adding a new row duplicating `AMZ` wouldn't be a correct test case.

Reply:
Agreed to directly return `null` on slack, pending change